src/structures/Sticker.js

```diff
@@ -0,0 +1,93 @@
+'use strict';
+
+const Base = require('./Base');
+const { StickerFormatTypes } = require('../util/Constants');
+const Snowflake = require('../util/Snowflake');
+
+/**
+ * Represents a Sticker.
+ * @extends {Base}
+ */
+class Sticker extends Base {
+  constructor(client, sticker) {
+    super(client);
+    /**
+     * The ID of the sticker
+     * @type {Snowflake}
+     */
+    this.id = sticker.id;
+
+    /**
+     * The ID of the sticker's image
+     * @type {string}
+     */
+    this.asset = sticker.asset;
+
+    /**
+     * The description of the sticker
+     * @type {string}
+     */
+    this.description = sticker.description;
+
+    /**
+     * The format of the sticker
+     * @type {string}
```

Comment:
The `StickerFormatTypes` typedef should be used here
```suggestion
     * @type {StickerFormatTypes}
```